cgmanifest.json

```diff
@@ -3200,6 +3380,266 @@
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXpm",
+          "version": "3.5.12",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libXpm-3.5.12.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXrandr",
+          "version": "1.5.1",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libXrandr-1.5.1.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXrender",
+          "version": "0.9.10",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libXrender-0.9.10.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXres",
+          "version": "1.0.7",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libXres-1.0.7.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXScrnSaver",
+          "version": "1.2.2",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libXScrnSaver-1.2.2.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libxshmfence",
+          "version": "1.2",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libxshmfence-1.2.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXt",
+          "version": "1.1.5",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libXt-1.1.5.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXtst",
+          "version": "1.2.3",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libXtst-1.2.3.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXv",
+          "version": "1.0.11",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libXv-1.0.11.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXvMC",
+          "version": "1.0.10",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libXvMC-1.0.10.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXxf86dga",
+          "version": "1.1.4",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libXxf86dga-1.1.4.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXxf86vm",
```

Comment:
The `libxcrypt` right after this one is no longer in alphabetical order.

Reply:
thanks, fixed